### DISCUSSION TITLE
game won't boot up without this mod anymore?

### DISCUSSION THREAD

**Discussion by poisax** [jbredwards/Fluidlogged-API#146] [General]
so while testing to see which mod/combination is causing issues i removed this one among others and now the game will boot just fine with the mod installed but as soon as i remove it the game fails to boot up so bad that it crashes prismmc...
i wasn't sure to list this as an issue as i'm probably just missing something, but in any case, [here's a log](https://pastebin.com/zC7WQtCw)

**Comment by embeddedt** [comment]
Friendly Chests needs Fluidlogged API to work.

**Reply by poisax** [reply]
i'm an idiot, thank you very much
